docs/release/trg-4/trg-4-02.md

```diff
@@ -23,6 +20,9 @@ As Eclipse Tractus-X project, we don't have automated processes for publishing c
 DockerHub is running container scans for all [official images](https://docs.docker.com/trusted-content/official-images/)
 and is publishing the scans result in the [docker-library/repo-info repository](https://github.com/docker-library/repo-info).
 
+Utilizing official Dockerhub images from DockerHub is advantageous due to its role as a trusted repository of curated and verified open source containers.
```

Comment:
Just a small typo to use similar format for `DockerHub` across the whole site.

```suggestion
Utilizing official DockerHub images from DockerHub is advantageous due to its role as a trusted repository of curated and verified open source containers.
```

Reply:
thanks @almadigabor I actually didn't want to duplicate Dockerhub word in the sentence so good finding, removed one.